### ISSUE TITLE
508-defect-1 ❗ Launchblocker [FOCUS MANAGMENT]: There MUST not be any keyboard traps

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8792]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

 In the keyboard test, I got stuck on the Search button. When I tabbed, I went to the content below, and it took a long time to get to the search results, which was disorienting as a sighted, keyboard and screen reader user. 

This appears to be a web performance issue, related to how long it takes for the results to render. In the interim, since focus is still on the Search button — instead of the loading indicator, and then the h2, it skips the search results.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to navigate from the Search button to the search results.

**Comment by kelsonic** [comment]
The desired behavior I've heard is that after searching, focus changes to search results header. When tabbing, however, you should be able to go to the items underneath the search button. Moving to validate as I believe this is moot: https://staging.va.gov/education/yellow-ribbon-participating-schools/

**Comment by jenstrickland** [comment]
The desired behavior is that focus would move to the loading indicator before the h2, so that focus doesn't go to the items beneath while the search results are being retrieved.

**Comment by kelsonic** [comment]
Awesome! It does that:

![image](https://user-images.githubusercontent.com/12773166/81323578-d4cacd00-9052-11ea-8fb5-34fce7e83c8e.png)

**Comment by jenstrickland** [comment]
Just validated that this issue has been addressed as of this afternoon.
